_posts/2017-05-30-database-encryption.markdown

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: "Database Encryption"
+date: 2017-05-30 11:03
+comments: true
+author: ashkan
+categories: [Database, Encryption, Security]
+---
+
+After examining the data stored in one of our systems, we realized it can potentially include some sensitive user data. To reduce the number of people that are technically able to access the data and reduce the risks associated with a potential data theft, we decided to encrypt data getting persisted in the database.
```

Comment:
_can_, _potentially_, _some_ all weaken the statement. I'd just say "...it might include sensitive user data."

"...decided to encrypt certain database fields."